src/app/components/demo-components.service.ts

```diff
@@ -66,6 +66,14 @@ export class SkyDemoComponentsService {
         url: '/components/checkbox',
         getCodeFiles: () => this.getDemoFiles('SkyCheckboxDemoComponent')
       },
+      {
+        name: 'Code block',
+        icon: 'code',
+        // tslint:disable-next-line
+        summary: `The code block component renders a formatted block of code with syntax highlighting.`,
```

Comment:
I'd try to make this match the opening sentence of the overview paragraph in the docs: "The code block component formats a code block with syntax highlighting." (Assuming that you implement the changes I suggested.)